app/containers/RegistrationLoginButtons.tsx

```diff
@@ -29,10 +29,11 @@ export const RegistrationLoginButtonsComponent: React.FunctionComponent<Props> =
     </>
   ) : (
     <>
-      <p>The due date for CIIP application forms has passed.</p>
+      <p>In 2022, CIIP applications will be accepted from May 24 to June 30.</p>
```

Comment:
Do we want this to be displayed based on the configured reporting periods?

Reply:
That's a great idea, we can suggest that as a feature for the business and implement it in a separate PR to make sure we get this shipped this week